charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -49,6 +49,11 @@ stringData:
   K8S_CLUSTER_URL: {{ .Values.rhdhOperator.k8s.clusterUrl }}
   K8S_CLUSTER_TOKEN: {{ .Values.rhdhOperator.k8s.clusterToken }}
 {{- end }}
+{{- if .Values.argocd.enabled }}
```

Comment:
if we see this one going out of control, we can create a dedicated ticket for it.